src/Tools/AI Test Toolkit/Permissions/AITestToolKitRead.PermissionSet.al

```diff
@@ -0,0 +1,18 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+permissionset 149033 "AI Test Toolkit - Read"
```

Comment:
Any reason why all these permission objects in the test toolkit can't be inherent? This is a test tool.